.hashibot.hcl

```diff
@@ -0,0 +1,11 @@
+poll "closed_issue_locker" "locker" {
+    schedule = "0 50 13 * * *"
```

Comment:
Unfortunately a lot of providers are running at the same time, I would suggest picking a new time, a shortcoming of hashicorp (we want to avoid API limiting)
```
schedule = "0 10 5 * * *"
```